cmd/application_stop.go

```diff
@@ -23,6 +26,18 @@ var applicationStopCmd = &cobra.Command{
 			panic("unreachable") // staticcheck false positive: https://staticcheck.io/docs/checks#SA5011
 		}
 
+		if applicationName == "" && applicationNames == "" {
+			utils.PrintlnError(fmt.Errorf("use neither --application \"<app name>\" nor --applications \"<app1 name>, <app2 name>\""))
```

Comment:
_typo_: wording => `use either --application \"<app name>\" or --applications \"<app1 name>, <app2 name>\" but not both at the same time`